src/clj/fluree/db/json_ld/migrate/sid.cljc

```diff
@@ -33,98 +33,98 @@
   (let [{:keys [namespaces namespace-codes]} @(:mapping ns-mapping)]
     (assoc db :namespaces namespaces, :namespace-codes namespace-codes)))
 
-(defn merge-commit
-  "Process a new commit map, converts commit into flakes, updates
-  respective indexes and returns updated db"
-  [conn {:keys [alias] :as db} [commit _proof]]
-  (go-try
-    (let [db-address (-> commit
-                         (get-first const/iri-data)
-                         (get-first-value const/iri-address))
-          _          (log/info "Migrating commit at address:" db-address)
-          db-data    (<? (reify/read-db conn db-address))
-          t-new      (reify/db-t db-data)
-          ns-mapping (db->namespace-mapping db)
+;; (defn merge-commit
+;;   "Process a new commit map, converts commit into flakes, updates
+;;   respective indexes and returns updated db"
+;;   [conn {:keys [alias] :as db} [commit _proof]]
+;;   (go-try
+;;     (let [db-address (-> commit
+;;                          (get-first const/iri-data)
+;;                          (get-first-value const/iri-address))
+;;           _          (log/info "Migrating commit at address:" db-address)
+;;           db-data    (<? (reify/read-db conn db-address))
+;;           t-new      (reify/db-t db-data)
+;;           ns-mapping (db->namespace-mapping db)
 
-          assert           (reify/db-assert db-data)
-          asserted-flakes  (reify/assert-flakes ns-mapping t-new assert)
-          retract          (reify/db-retract db-data)
-          retracted-flakes (reify/retract-flakes ns-mapping t-new retract)
-          db*              (set-namespaces db ns-mapping)
+;;           assert           (reify/db-assert db-data)
+;;           asserted-flakes  (reify/assert-flakes ns-mapping t-new assert)
+;;           retract          (reify/db-retract db-data)
+;;           retracted-flakes (reify/retract-flakes ns-mapping t-new retract)
+;;           db*              (set-namespaces db ns-mapping)
 
-          {:keys [previous issuer message data] :as commit-metadata}
-          (commit-data/json-ld->map commit db*)
+;;           {:keys [previous issuer message data] :as commit-metadata}
+;;           (commit-data/json-ld->map commit db*)
```

Comment:
Should we delete all this commented-out code?

Reply:
I decided to comment it out for now because we will very soon have to re-write this code, so it won't stay this way for long, and because it won't be a wholesale rewrite since the underlying logic of the migration will be the same. There's just a few indexing and db loading apis that have/will change.